requirements-dev.txt

```diff
@@ -34,3 +34,6 @@ bump2version==0.5.8
 
 # Test support
 matrix-synapse==0.33.9
+
+# temporary
+pytest-repeat==0.7.0
```

Comment:
what is the difference with pytest-flaky? (we had problems with it in the past, not sure if was because of gevent, the tests, or the library)

Reply:
This dependency was only added for a test build, not intended to be merged.